app/page.tsx

```diff
@@ -22,8 +23,7 @@ const Home = () => {
         </Box>
       </Box>
       <Box
-        w={{ base: "base", md: "741px", xl: "xl" }}
-        h={{ base: "323px", md: "411px", xl: "462px" }}
+        w={{ base: "base", xl: "xl" }}
```

Comment:
is it confirmed we won't be using the width and height from the current spec?

Reply:
Something along these lines, but you probably need to adjust at least the padding between:
```tsx
<Box
  w={{ base: "base", md: "741px", xl: "xl" }}
  p={{
    base: "23px 24px 8px 24px",
    md: "37px 27px 8px 26px",
    xl: "50px 128px 8px 127px",
  }}
  m="auto"
>
  <Report />
</Box>
<Box
  w={{ base: "base", md: "741px", xl: "xl" }}
  h={{ base: "323px", md: "411px", xl: "462px" }}
  p={{
    base: "23px 24px 27px 24px",
    md: "37px 27px 28px 26px",
    xl: "50px 128px 40px 127px",
  }}
  m="auto"
>
  <Box
    h="100%"
    border="1px solid"
    borderColor="grey.400"
    borderRadius="8px"
    boxShadow="0px 2px 4px -1px #0000000F, 0px 4px 6px -1px #0000001A"
    overflow="hidden"
  >
    <Map />
  </Box>
```

Reply:
I've left the map out of the report and kept its width and height parameters as they were originally done.